apps/vr-tests/src/stories/ButtonConverged.stories.tsx

```diff
@@ -18,18 +18,41 @@ storiesOf('Button Converged', module)
       {story()}
     </Screener>
   ))
-  .addStory('Default', () => <Button>Hello, world</Button>, { rtl: true })
+  .addStory('Default', () => <Button>Hello, world</Button>, {
+    rtl: true,
+    includeHighContrast: true,
+    includeDarkMode: true,
+  })
   .addStory('Circular', () => <Button circular>Hello, world</Button>)
   .addStory('Outline', () => <Button outline>Hello, world</Button>)
-  .addStory('Primary', () => <Button primary>Hello, world</Button>)
-  .addStory('Subtle', () => <Button subtle>Hello, world</Button>)
-  .addStory('Transparent', () => <Button transparent>Hello, world</Button>)
-  .addStory('Disabled', () => <Button disabled>Hello, world</Button>)
-  .addStory('Outline Disabled', () => (
-    <Button outline disabled>
-      Hello, world
-    </Button>
-  ))
+  .addStory('Primary', () => <Button primary>Hello, world</Button>, {
+    includeHighContrast: true,
+    includeDarkMode: true,
+  })
+  .addStory('Subtle', () => <Button subtle>Hello, world</Button>, {
+    includeHighContrast: true,
+    includeDarkMode: true,
+  })
+  .addStory('Transparent', () => <Button transparent>Hello, world</Button>, {
+    includeHighContrast: true,
+    includeDarkMode: true,
+  })
+  .addStory('Disabled', () => <Button disabled>Hello, world</Button>, {
+    includeHighContrast: true,
+    includeDarkMode: true,
+  })
+  .addStory(
+    'Outline Disabled',
+    () => (
+      <Button outline disabled>
+        Hello, world
+      </Button>
+    ),
+    {
+      includeHighContrast: true,
+      includeDarkMode: true,
+    },
+  )
```

Comment:
We should either add it to all the '[x] Disabled` stories or to none, right now it is only being added to `Outline Disabled`.